src/Http/Controllers/Controller.php

```diff
@@ -469,6 +448,34 @@ private function is_animated_gif($filename)
         return $frames > 1;
     }
 
+    public function validateBread($request, $data)
```

Comment:
Why did you move this method?  It does nothing except complicate the diff.  Please either provide a good reason or move it back.

Reply:
I did move it back , but in an amend commit by wrong